ompi/mpi/fortran/use-mpi-f08/abort_f08.F90

```diff
@@ -3,10 +3,14 @@
 ! Copyright (c) 2010-2012 Cisco Systems, Inc.  All rights reserved.
 ! Copyright (c) 2009-2012 Los Alamos National Security, LLC.
 !                         All rights reserved.
-! Copyright (c) 2018      Research Organization for Information Science
+! Copyright (c) 2018-2019 Research Organization for Information Science
 !                         and Technology (RIST).  All rights reserved.
 ! $COPYRIGHT$
 
+#if OMPI_BUILD_MPI_PROFILING
+#define MPI_Abort_f08 PMPI_Abort_f08
+#endif
```

Comment:
Just curious: why not put all these `#define`'s in a single `.h` file and `#include` it in all the `.F90` files?

That would also cover you just in case any of these `.F90` files invoke any `MPI_Foo` subroutines -- they would get automatically transmorgraphied into `PMPI_Foo` (I don't think that we actually do that, but it's a nice thought 😉).

I mention this because it's just slightly less code in each of these hundreds of `.F90` files, and it's one central place to update if we need to update the #defines (e.g., if we need to add `ts` to the end of the symbol name for when we support `type(*), dimension(..)`).

Reply:
it does make sense and I will do that, thank!